ingestion/functions/parsing/new_zealand/new_zealand.py

```diff
@@ -62,13 +62,13 @@ def convert_demographics(entry):
     If age is listed as 90+, setting age range as between 90 and 120.
     '''
     demo = {}
-    if age := entry[_AGE]:
+    if (age := entry[_AGE]) != "NA":
```

Comment:
we now enter the true branch if age is None, and (still) raise `KeyError` if `_AGE not in entry`, are these both intended?

Reply:
_AGE should always be in entry, it can be NA or a valid value